lua/org-roam/database.lua

```diff
@@ -94,6 +98,10 @@ function M:files_path()
     return self.__directory
 end
 
+function M:org_files()
```

Comment:
Since this is a publicly-exposed function, would you mind giving it a Luadoc comment? For example, it would be good to note that the files returned are globs and not necessarily individual files or directories.